src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```

этот метод должен возвращать имегнно экземпляр класса Ball, а не String